test/test_bundle_importer.js

```diff
@@ -191,26 +213,70 @@ describe('Bundle Importer', function() {
         addMachines: true,
         deploy: true,
         addUnit: true,
-        addRelation: true
+        addRelation: true,
+        setAnnotations: true
       });
     });
-
-    it('does not fail if unknown record type', function() {
+    it('properly sorts a recordSet', function() {
```

Comment:
can you please leave a blank line between tests? This makes it harder to find the start/end of them.